src/main/java/edu/kit/hci/soli/controller/BookingsController.java

```diff
@@ -109,6 +109,18 @@ public String newBooking(
         return "create_booking";
     }
 
+    @GetMapping("/bookings/view/{eventId}")
```

Comment:
maybe add the room id just like the other endpoints

Reply:
Wenn man im Kalender einen Termin anklickt ist man schon im Raum